src/Transports/MassTransit.KafkaIntegration/KafkaIntegration/Configuration/Cached.cs

```diff
@@ -0,0 +1,60 @@
+namespace MassTransit.KafkaIntegration.Configuration
+{
+    using System;
+    using System.Collections.Generic;
+    using Confluent.Kafka;
+
+
+    static class Cached
```

Comment:
So, the outer Cached class isn't needed in this case, since you aren't really dealing with the weirdness. So it could be removed.

Reply:
It isn't really a cache anyway, it's just a dictionary of known types.